lib/main.js

```diff
@@ -120,6 +121,15 @@ module.exports.runBridge = Promise.coroutine(function*(port, config, reg) {
     }
 
     // run the bridge
-    var ircBridge = new IrcBridge(config, reg);
+    ircBridge = new IrcBridge(config, reg);
     yield ircBridge.run(port);
 });
+
+module.exports.killBridge = function() {
+    if (!ircBridge) {
+        log.info('killBridge(): No bridge running');
+        return Promise.resolve({});
```

Comment:
`Promise.resolve()` would be clearer, else it looks like something is going to look at the result of this.